lib/config/validation.spec.ts

```diff
@@ -391,10 +392,11 @@ describe('config/validation', () => {
       expect(errors).toMatchSnapshot();
     });
 
-    it('errors if no regexManager matchStrings', async () => {
+    it('errors if regexManager has empty fileMatch', async () => {
```

Comment:
Why do you change these existing tests?

Reply:
The test name is misleading because the expected error is for a empty fileMatch arrar